### ISSUE TITLE
azurerm_virtual_network_gateway argument vpn_client_protocol not recognized

### ISSUE THREAD

**Issue by Phydeauxman** [terraform-providers/terraform-provider-azurerm#1485]
### Terraform Version

Terraform v0.11.7
+ provider.azurerm v1.8.0

### Affected Resource(s)

azurerm_virtual_network_gateway

### Terraform Configuration Files

```hcl
resource "azurerm_virtual_network_gateway" "sdvnet" {
  name                = "${var.project_ident}-${var.env_ident}-vnetgw"
  location            = "${data.azurerm_resource_group.core-rg.location}"
  resource_group_name = "${data.azurerm_resource_group.core-rg.name}"
  type                = "Vpn"
  vpn_type            = "RouteBased"
  active_active       = false
  sku                 = "VpnGw1"

  ip_configuration {
    name                          = "vnetGatewayConfig"
    public_ip_address_id          = "${azurerm_public_ip.pubip1.id}"
    private_ip_address_allocation = "Dynamic"
    subnet_id                     = "${azurerm_subnet.sub1.id}"
  }

  vpn_client_configuration {
    address_space       = ["192.168.200.0/24"]
    vpn_client_protocol = ["SSTP", "IkeV2"]

    root_certificate {
      name = "${var.project_ident}-${var.env_ident}-rootcert"

      public_cert_data = <<EOF
<Removed_Cert>EOF
    }
  }
}
```

### Expected Behavior

Virtual Network Gateway should be created with both protocols enabled

### Actual Behavior

The following error is generated:
`Error: azurerm_virtual_network_gateway.sdvnet: vpn_client_configuration.0: invalid or unknown key: vpn_client_protocol`

### Steps to Reproduce

1. `terraform apply`

### References

#946

**Comment by dcaro** [comment]
Thanks @Phydeauxman actually the key should be vpn_client_protocols. Let me submit a change to the doc.

**Comment by dcaro** [comment]
Submitted the following PR to update the documentation #1486

**Comment by Phydeauxman** [comment]
@dcaro thanks for the quick reply...adding the s did the trick.

**Comment by katbyte** [comment]
fixed by #1486
